<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VentanaLogin</class>
 <widget class="QMainWindow" name="VentanaLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>430</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Avenir</family>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string notr="true">Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#VentanaLogin{
background-color: rgb(49, 50, 50);
}
#label_login{
	color: rgb(255, 255, 255);
}
#label_usuario{
	color: rgb(255, 255, 255);
}
#label_contrasena{
	color: rgb(255, 255, 255);
}
#line_usuario{
color: rgb(97, 99, 99);
}
#line_contrasena{
color: rgb(97, 99, 99);
}
#button_iniciar{
	color: rgb(250, 255, 255);
	background-color: rgb(168, 0, 16);
	border: none;
}
#button_iniciar:hover{
	color: rgb(168, 0, 16);
	background-color: rgb(255, 255, 255);
	border: none;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="line_usuario">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_contrasena">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_usuario">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Usuario</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_contrasena">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Contraseña</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_login">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir</family>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="button_iniciar">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>330</y>
      <width>300</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Avenir</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>Iniciar Sesión</string>
    </property>
   </widget>
   <zorder>label_usuario</zorder>
   <zorder>line_usuario</zorder>
   <zorder>line_contrasena</zorder>
   <zorder>label_contrasena</zorder>
   <zorder>label_login</zorder>
   <zorder>button_iniciar</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
